src/views/exportJarFile.ts

```diff
@@ -0,0 +1,335 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import { EOL, platform } from "os";
+import { basename, extname, join } from "path";
+import { CancellationToken, commands, Extension, extensions, MessageItem, MessageOptions,
+         ProgressLocation, QuickInputButtons, Uri, window, workspace, WorkspaceFolder } from "vscode";
+
+import { isStandardServerReady } from "../extension";
+import { Jdtls } from "../java/jdtls";
+import { INodeData } from "../java/nodeData";
+
+import { buildWorkspace } from "./build";
+import { QuickPickNode } from "./quickPickNode";
+import { WorkspaceNode } from "./workspaceNode";
+
+const SOLVE_PROJECT = "solve project";
+const SOLVE_MAINMETHOD = "solve mainmethod";
+const GENERATE_JAR = "generate jar";
+const FINISH = "finish";
+
+export class ExportJarFile {
```

Comment:
Why not just make all the member functions simply utility functions? The class here is no needed